csuntechlab/iSTART
# STR-132 -- FE: Implement Information Slides #3, #4, #5, #10, #17 into Alcohol Module

# Description
Updated info template to reflect new JSON, implemented new info slides.

# Testing
1. Login as `cyndee.bellamy` or `april.feldman`.
2. Visit alcohol module.
3. Ensure all implemented slides reflect the alcohol module provided.
4. Make sure the images load and are properly positioned in the right place.
5. Check the slide content to make sure it makes sense and engrish.
